baselines/audioworklet.generated.d.ts

```diff
@@ -38,6 +38,15 @@ interface MessageEventInit<T = any> extends EventInit {
     source?: MessageEventSource | null;
 }
 
+interface PermissionNameMap {
```

Comment:
I'm not sure what's special about this file that this type is being generated here but `PermissionName` is not and i'm not sure how to control this.